classes/OpeningHours/Module/Shortcode/IsOpen.php

```diff
@@ -81,12 +83,45 @@ public function shortcode ( array $attributes ) {
 
     $attributes['is_open'] = $isOpen;
     $attributes['classes'] .= ($isOpen) ? $attributes['open_class'] : $attributes['closed_class'];
-    $attributes['text'] = ($isOpen) ? $attributes['open_text'] : $attributes['closed_text'];
+
+    // If the attribute show_holiday is enabled
+    $closedText = $attributes['closed_text'];
+    if ($attributes['show_closed_holidays']) {
+        $holidaysList = $this->getTodaysHolidaysCommaSeperated($todayData);
+
+        $closedText = sprintf(
```

Comment:
I think it would be better to not reassign `$closedText` when there are no holidays and not `sprintf`ing it again with `$holidayList`

Reply:
Agreed